pkg/naming/constants.go

```diff
@@ -40,13 +40,23 @@ const (
 	ScyllaServiceTypeMember   ScyllaServiceType = "member"
 )
 
+type ScyllaIngressType string
+
+const (
+	ScyllaIngressTypeDirect ScyllaIngressType = "direct"
+	ScyllaIngressTypeAny    ScyllaIngressType = "any"
```

Comment:
although the `any` is the dns name, `NodeLB` or `Cluster` types might be more descriptive here. possibly `AnyNode`